sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/CosmosEncryptionContainer.java

```diff
@@ -359,6 +365,60 @@ public CosmosBatchResponse executeCosmosBatch(
         return this.blockBatchResponse(this.cosmosEncryptionAsyncContainer.executeCosmosBatch(cosmosBatch, requestOptions));
     }
 
+    /**
+     * Executes flux of operations in Bulk.
```

Comment:
There is no flux in sync container, please fix documentation for these APIs.

Reply:
Fixed